ci/cloudbuild/triggers/publish-docs-compute-pr.yaml

```diff
@@ -0,0 +1,16 @@
+filename: ci/cloudbuild/cloudbuild.yaml
+github:
+  name: google-cloud-cpp
+  owner: googleapis
+  pullRequest:
+    branch: ^main$
+    commentControl: COMMENTS_ENABLED_FOR_EXTERNAL_CONTRIBUTORS_ONLY
+name: publish-docs-compute-pr
+substitutions:
+  _BUILD_NAME: publish-docs
+  _DISTRO: fedora-37-cmake
+  _TRIGGER_TYPE: pr
+  _LIBRARIES: compute
```

Comment:
Aside, not so related to this PR:

I am always wanting a way to build `bigtable` only (for example) while developing. I end up manually commenting out build code in basically every new dev branch I make.

It would be interesting to see this `_LIBRARIES` variable get spread across all of our builds. So I could edit just that field in `clang-tidy-pr.yaml` locally.

I might be alone though. This is just a thought. Maybe someone will be inspired.